pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -964,9 +972,9 @@ type FilesystemSpec struct {
 	// +nullable
 	MetadataPool PoolSpec `json:"metadataPool"`
 
-	// The data pool settings
-	// +nullable
-	DataPools []PoolSpec `json:"dataPools"`
+	// The data pool settings, with optional predefined pool name.
+	// +optional
```

Comment:
I think this might be a carryover from the previous design?

Reply:
Looks like, i'll return nullable as it was before